c7n/resources/sagemaker.py

```diff
@@ -342,6 +380,7 @@ class resource_type(TypeInfo):
 @NotebookInstance.action_registry.register('tag')
 @SagemakerJob.action_registry.register('tag')
 @SagemakerTransformJob.action_registry.register('tag')
+@SagemakerHyperParameterTuningJob.action_registry.register('tag')
```

Comment:
If resource group tagging apis are supported on this resource, you would not need tag/remove-tag actions separately